src/nav/components.yml

```diff
@@ -54,7 +54,7 @@
           title: Input
         - id: /components/radio
           title: Radio
-        - id: /components/range
+        - id: https://garden.zendesk.com/react-components/forms/#range
```

Comment:
This needs to use the `zendeskgarden.github.io` domain, otherwise the link will be broken on DNS switch.